code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -223,6 +223,9 @@
 		holder = null
 	..()
 
+/datum/reagent/proc/steve(var/obj/item/weapon/reagent_containers/food/drinks/drinkingglass/D)
```

Comment:
As much as I liked steve, he needs a better name. No suggestions from me, though.

Reply:
steve has changed his name to `handle_special_behavior`.